### ISSUE TITLE
Optimize VMs for Sidekiq jobs

### ISSUE THREAD

**Issue by acozine** [pulibrary/figgy#4760]
Sidekiq (at least when running under MRI, see [FAQ entry](https://github.com/mperham/sidekiq/wiki/Problems-and-Troubleshooting#why-isnt-sidekiq-using-all-of-the-available-cores)) makes best usage of available resources when the number of processes matches the number of CPUs. Configure VMs and/or Sidekiq to match this pattern, monitor performance.

3 VMs currently run Sidekiq jobs:
lib-proc6: 4 cores, 5 processes
lib-proc8: 8 cores, 5 processes
lib-proc9: 8 cores, 5 processes

**Comment by acozine** [comment]
@tpendragon good to know. I'll start by increasing the cores on lib-proc6, which is the VM that keeps choking. Once all three VMs have the same resources, I'll think about next steps.

**Comment by acozine** [comment]
I've updated `lib-proc6` so it now has 8 cores. It still runs 5 processes. 

Looking at our Sidekiq setup, there are three more Sidekiq VMs in the other VMCluster. Stats on those three:

lib-proc7: 2 cores, 5 processes
lib-proc10: 2 cores, 5 processes
lib-proc11: 2 cores, 5 processes

All 6 VMs are configured to run jobs from all queues (high, default, low, and super-low). If we continue to see issues around background jobs, we might need to add more resources to the other 3 VMs and/or configure the less-beefy machines to run the lower-priority jobs.

For now, however, we have stability with the additional resources on lib-proc6. Closing with the knowledge that we may need to reopen in future . . .
